apps/avifdec.c

```diff
@@ -36,6 +36,9 @@ static void syntax(void)
     printf("    -u,--upsampling U : Chroma upsampling (for 420/422). automatic (default), fastest, best, nearest, or bilinear\n");
     printf("    -r,--raw-color    : Output raw RGB values instead of multiplying by alpha when saving to opaque formats\n");
     printf("                        (JPEG only; not applicable to y4m)\n");
+    printf("    --index           : When decoding an image sequence or progressive image, specify which frame index to decode (Default: 0)\n");
+    printf("    --progressive     : Enable progressive AVIF processing. If a progressive image is encountered, avifdec will use\n");
+    printf("                        --index to choose which variant to decode (in progressive order).\n");
```

Comment:
Should we change "variant" to "layer" or "frame"?

Reply:
Layer is probably better here.